src/render.jl

```diff
@@ -60,16 +63,18 @@ function push!(ents::Entries, modulename::Module, obj, ent::AbstractEntry)
 end
 
 file"docs/save.md"
-function save(file::AbstractString, modulename::Module; args...)
-    config = Config(; args...)
+function save(file::AbstractString, modulename::Module, config::Config; args...)
+    isempty(args) || update_config!(config, Dict(args))
```

Comment:
This will mutate `config` which probably isn't the behaviour we want? Constructing a new `Config` by merging `args` into `config` would be better. Possibly just `update_config!(deepcopy(config), Dict(args))`?